### PR TITLE
Fix insults terminal corruption

### PR DESCRIPTION
**Pull request by markrwilliams**
http://twistedmatrix.com/trac/ticket/8803

I went a little crazy with the tests and did find some Python-3 related problems.

### CODE REVIEW

File: src/twisted/conch/test/test_insults.py [hunk 1]
```diff
@@ -7,10 +8,14 @@
 from twisted.test.proto_helpers import StringTransport
 
 from twisted.conch.insults.insults import ServerProtocol, ClientProtocol
-from twisted.conch.insults.insults import CS_UK, CS_US, CS_DRAWING, CS_ALTERNATE, CS_ALTERNATE_SPECIAL
+from twisted.conch.insults.insults import CS_UK, CS_US, CS_DRAWING, CS_ALTERNATE, CS_ALTERNATE_SPECIAL, BLINK, UNDERLINE
```
Comment (hawkowl) [inline src/twisted/conch/test/test_insults.py]: too long line here

File: src/twisted/conch/test/test_insults.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (hawkowl) [inline src/twisted/conch/test/test_insults.py]: spurious newline here?

File: src/twisted/conch/topfiles/8803.bugfix [hunk 3]
```diff
@@ -0,0 +1 @@
+A Conch server no longer corrupts a client's display when attempting to set the cursor position, and insults' ServerProtocol's ECMA-48 terminal manipulation works on Python 3.
```
Comment (hawkowl) [inline src/twisted/conch/topfiles/8803.bugfix]: We should have a fqdn here, eg "twisted.conch servers using twisted.conch insults no longer corrupts..."

File: src/twisted/conch/test/test_insults.py (hunk 2 above)
Comment (glyph) [reply src/twisted/conch/test/test_insults.py]: Looks like a separation between import sections (twisted, stdlib) to me.

**Comment by glyph** [comment]
This looks _great_ to me; just restarting a couple of builds to see if we can get that checkmark to turn green…

**Pull request merged**